### PR TITLE
Add Calibre

### PR DESCRIPTION
**Pull request by 85pando**
Calibre is a program to manage your ebook library. It has several usefull command-line tools.

### CODE REVIEW

**Comment by lord63** [comment]
Don't forget to update the index.json

**Comment by igorshubovych** [comment]
@rprieto,

I noticed there are 2 different ways of listing multiple related arguments.

In this and some other pages several arguments are listed in one token:
`calibredb add {{file1, file2, …}}`
`zip -r {{compressed.zip}} {{/path/to/dir1 /path/to/dir2 /path/to/file}}`

On other pages arguments are listed one by one, 
`diff {{file1}} {{file2}}`
`zpool create {{poolname}} mirror {{disk1}} {{disk2}}`

What is the preferred way?

**Comment by 85pando** [comment]
Given [this comment from rprieto](https://github.com/tldr-pages/tldr/pull/306#issuecomment-152023509) i think it is clear it should be one command per example.

**Comment by igorshubovych** [comment]
This commit has one command per example as it is mentioned in Contributing guidelines.

My question was regarding the way tokens are listed in a single example.

`command {{file1, file2}}`

vs

`command {{file1}} {{file2}}`

**Comment by 85pando** [comment]
Sorry, too early (-;

> User-provided values should use the `{{token}}` syntax, to allow clients to highlight them. For example: `tar cf {{file}}`

So in my understanding, the token is just needed for highlighting values. For the `diff` example at least that is a different case, as both values are needed. In `calibredb add` you need one file but more files are possible.

File: pages/common/calibredb.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+`calibredb list`
+
+- Search for ebooks displaying additional information.
+
+`calibredb list --search {{search-term}}`
+
+- Search for just ids of ebooks.
+
+`calibredb search {{search term}}
+
```
Comment (rprieto) [inline pages/common/calibredb.md]: Just a missing backtick here

File: pages/common/ebook-convert.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# ebook-convert
+
+> Can be used to convert ebooks between common formats, e.g., pdf, epub and mobi.
+> Part of the Calibre ebook library tool.
+
+- Convert an ebook into another format.
+
+`ebook-convert {{source}} {{destination}}
```
Comment (rprieto) [inline pages/common/ebook-convert.md]: and here :)

**Comment by igorshubovych** [comment]
@rprieto,

Can we automate checking missing backticks?

**Pull request merged**